python/paddle/incubate/hapi/metrics.py

```diff
@@ -190,7 +190,7 @@ class Accuracy(Metric):
             
         model.prepare(
             optim,
-            loss_function=CrossEntropy(average=False),
+            loss_function=CrossEntropyLoss(),
```

Comment:
Might it be `CrossEntropyLoss(reduction="sum")`

Reply:
Here is only an example, for shorter code, not add `reduction="sum"`